.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
not valid json

Reply:
this was generated but sure, I will remove it , good point